.ci/pytorch/common_utils.sh

```diff
@@ -202,7 +202,7 @@ function install_torchrec_and_fbgemm() {
 
 function clone_pytorch_xla() {
   if [[ ! -d ./xla ]]; then
-    git clone --recursive --quiet https://github.com/pytorch/xla.git
```

Comment:
Why was `--quiet` removed by the script? Is this expected or does it imply a bug in the script?

Reply:
no ``quiet`` option usually helps with debug, if there are some errors or anything like this. It will be easier to spot. However I don't think its does any functional change. We applied this code for a few releases already. I don't think its a bug.